endpoints_management/control/check_request.py

```diff
@@ -143,7 +144,7 @@ def convert_response(check_response, project_id):
     api_key_is_bad: indicates that a given api_key is bad
 
     Args:
-       check_response (:class:`endpoints_management.gen.servicecontrol_v1_messages.CheckResponse`):
+       check_response (:class:`endpoints_management.gen.servicecontrol_v1_sc_messages.CheckResponse`):
```

Comment:
I don't think this change is quite correct.

Reply:
Done.